addons/mail/static/src/components/chat_window_header/chat_window_header.scss

```diff
@@ -54,6 +54,11 @@
     text-overflow: ellipsis;
 }
 
+.o_Clickable_ChatWindowHeaderName:hover {
+    font-weight: bold;
+    text-decoration: underline;
```

Comment:
`$link-hover-decoration` ?

Reply:
@seb-odoo $link-hover-decoration is not working, it is taking None value instead of underline.

Reply:
This means we don't do underline of hover in Odoo and the spec should be challenged. This goes against our own styling guideline to do it.

Reply:
@seb-odoo Done the changes.